src/universal/modules/teamDashboard/containers/Team/Team.js

```diff
@@ -0,0 +1,29 @@
+import React, {Component, PropTypes} from 'react';
+import Team from 'universal/modules/teamDashboard/components/Team/Team';
+import requireAuth from 'universal/decorators/requireAuth/requireAuth';
+import {connect} from 'react-redux';
+
+const mapStateToProps = (state, router) => {
```

Comment:
the 2nd arg is `props`, not router... but we should kill this pattern with fire. If the view layer changes based on the location, that should be handled by the router, not inside our components. Will write details soon